docs/content/preview/develop/quality-of-service/write-heavy-workloads.md

```diff
@@ -23,66 +24,59 @@ The idea is to slow down incoming writes to the speed that the database can hand
 
 ### Limited disk IOPS or bandwidth
-In many cloud environments, the disk IOPS and/or the network bandwidth is rate-throttled. Holistically, if there are such disk constraints (limits on iops / bandwidth), that results in a back-pressure across all disk writes in the system, and would therefore manifest as one of the above conditions.
 
+In many cloud environments, the disk IOPS and/or the network bandwidth is rate-throttled. Holistically, if there are such disk constraints (limits on IOPS / bandwidth), that results in a back-pressure across all disk writes in the system, and would therefore manifest as one of the above conditions.
 
 ## Write throttling triggers
 
-YugabyteDB defines the following triggers for throttling incoming writes.
+YugabyteDB defines the following triggers for throttling incoming writes:
 
 ### Stop writes trigger
 
 Stop writes trigger is activated in one of the following scenarios:
 
-* **Too many SST files:** The number of SST files exceeds the value determined by the flag `sst_files_hard_limit`, which defaults to 48. Once the hard limit is hit, no more writes are processed, all incoming writes are rejected.
+* **Too many SST files:** The number of SST files exceeds the value determined by the flag `sst_files_hard_limit`, which defaults to 48. After the hard limit is hit, no more writes are processed, all incoming writes are rejected.
+
+* **Memstores flushed too frequently:** This condition occurs if there are a large number of tables (or more accurately, a large number of tablets) all of which get writes. In such cases, the memstores are forced to flush frequently, resulting in too many SST files. In such cases, you can tune the total memstore size allocated. Total memstore size is the minimum of the two flags: `global_memstore_size_mb_max` (default value is 2GB) and `global_memstore_size_percentage` (defaults to 10% of total YB-TServer memory allocated. There are 2 different options for controlling how much memory is allocated to YB-TServer:
 
-* **Memstores flushed too frequently:**
-This condition occurs if there are a large number of tables (or more accurately, a large number of tablets) all of which get writes. In such cases, the memstores are forced to flush frequently, resulting in too many SST files. In such cases, you can tune the total memstore size allocated. Total memstore size is the minimum of the two flags: `global_memstore_size_mb_max` (default value is 2GB) and `global_memstore_size_percentage` (defaults to 10% of total YB-TServer memory allocated. There are 2 different options for controlling how much memory is allocated to YB-TServer:
   * By setting `default_memory_limit_to_ram_ratio` to control what percentage of total RAM on the instance the process should use
```

Comment:
* Set `default_memory_limit_to_ram_ratio` to control percentage of total RAM on the instance the process should use.
  * Specify an absolute value using `memory_limit_hard_bytes`. For example, to provide YB-TServer with 32GB of RAM, use `--memory_limit_hard_bytes 34359738368`.